test/acceptance/features/bindAppToService.feature

```diff
@@ -305,3 +306,180 @@ Feature: Bind an application to a service
+        Given Backend service "some-backend-service.v0.2.0" CSV is installed
+            """
+            apiVersion: operators.coreos.com/v1alpha1
+            kind: ClusterServiceVersion
+            metadata:
+                name: some-backend-service.v0.2.0
+            spec:
+                displayName: Some Backend Service
+                install:
+                    strategy: deployment
+                customresourcedefinitions:
+                    owned:
+                        - name: backservs.service.example.com
+                          version: v1
+                          kind: Backserv
+                          statusDescriptors:
+                            - description: Name of the Secret to hold the DB user and password
+                              displayName: DB Password Credentials
+                              path: secret
+                              x-descriptors:
+                              - urn:alm:descriptor:io.kubernetes:Secret
+                              - binding:env:object:secret:username
+                              - binding:env:object:secret:password
+                            - description: Name of the ConfigMap to hold the DB config
+                              displayName: DB Config Map
+                              path: configmap
+                              x-descriptors:
+                              - urn:alm:descriptor:io.kubernetes:ConfigMap
+                              - binding:env:object:configmap:db_host
+                              - binding:env:object:configmap:db_port
+            """
+        * The CRD "backservs.service.example.com" is present
+            """
+            apiVersion: apiextensions.k8s.io/v1beta1
+            kind: CustomResourceDefinition
+            metadata:
+                name: backservs.service.example.com
+            spec:
+                group: service.example.com
+                versions:
+                    - name: v1
+                      served: true
+                      storage: true
+                scope: Namespaced
+                names:
+                    plural: backservs
+                    singular: backserv
+                    kind: Backserv
+                    shortNames:
+                    - bs
+            """
+        * The CR "demo-backserv-cr-2" is present
+            """
+            apiVersion: service.example.com/v1
+            kind: Backserv
+            metadata:
+                name: demo-backserv-cr-2
+            status:
+                secret: csv-demo-secret
+                configmap: csv-demo-cm
+            """
+        * The CR "csv-demo-cm" is present
+            """
+            apiVersion: v1
+            kind: ConfigMap
+            metadata:
+                name: csv-demo-cm
+            data:
+                db_host: 172.72.2.0
+                db_port: "3306"
+            """
+        * The CR "csv-demo-secret" is present
```

Comment:
same above, it reads better like `Secret is present`.

Reply:
done